### PR TITLE
docs(examples): add postgres.js example

### PR DESCRIPTION
**Pull request by silvioprog**
## Bug

- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added
- [ ] Errors have a helpful link attached, see `contributing.md`

## Feature

- [ ] Implements an existing feature request or RFC. Make sure the feature request has been accepted for implementation before opening a PR.
- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added
- [ ] Documentation added
- [ ] Telemetry added. In case of a feature if it's used or not.
- [ ] Errors have a helpful link attached, see `contributing.md`

## Documentation / Examples

- [x] Make sure the linting passes by running `pnpm build && pnpm lint`
- [x] The "examples guidelines" are followed from [our contributing doc](https://github.com/vercel/next.js/blob/canary/contributing/examples/adding-examples.md)

### CODE REVIEW

File: examples/with-postgres/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+yarn create next-app --example with-postgres with-postgres-app
+```
+
+```bash
+pnpm create next-app --example with-postgres with-postgres-app
+```
+
+## Configuration
+
+### Install dependencies
```
Comment (balazsorban44) [inline examples/with-postgres/README.md:29]: This should already be done by the create next-app CLI, so probably safe to remove

File: examples/with-postgres/ley.config.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+const { parse } = require('pg-connection-string')
+
+const options = {}
+
+if (process.env.DATABASE_URL) {
+  const { host, database, user, password, ssl } = parse(
+    process.env.DATABASE_URL
+  )
+
+  process.env.PGHOST = host
```
Comment (balazsorban44) [inline examples/with-postgres/ley.config.js:11]: Did you mean to assign to `options` here?

File: examples/with-postgres/pages/_document.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { Html, Head, Main, NextScript } from 'next/document'
```
Comment (balazsorban44) [inline examples/with-postgres/pages/_document.tsx:1]: Seems to be unnecessary, let's remove this file.

**Review by balazsorban44 — CHANGES_REQUESTED** [review]
Thanks! Added some comments.

File: examples/with-postgres/README.md (hunk 1 above)
Comment (silvioprog) [reply examples/with-postgres/README.md:29]: Makes sense, removing...

File: examples/with-postgres/ley.config.js (hunk 2 above)
Comment (silvioprog) [reply examples/with-postgres/ley.config.js:11]: Yes yes, you are right. Refactoring...

File: examples/with-postgres/pages/_document.tsx (hunk 3 above)
Comment (silvioprog) [reply examples/with-postgres/pages/_document.tsx:1]: Removing...

**Comment by silvioprog** [comment]
Thanks a lot for the tips. Refactored as suggested! :)

**Comment by maxproske** [comment]
Cool! I'm excited to see the first postgres data fetching example get added. 🎉

**Comment by silvioprog** [comment]
Preview: [nextjs-demo.silvioprog.deta.app](https://dev-pelican-nextjsdemo.silvioprog.deta.app) 🔗

**Review by balazsorban44 — APPROVED** [review]

**Pull request merged**